### DISCUSSION TITLE
Number of stars in the Starstruck badge seems to be wrong

### DISCUSSION THREAD

**Discussion by realamirhe** [community/community#20577] [Other Feature Feedback, Questions, & Ideas] [answered]
It seems to be hardcoded "⭐️ 16 stars"

https://github.com/realamirhe?tab=achievements&achievement=starstruck
https://github.com/realamirhe/immer-store has 56 starts

https://github.com/aaronchall?tab=achievements&achievement=starstruck
https://github.com/aaronchall/pythontrainer has 47 stars

**Comment by shaido987** [comment, marked as answer]
Yes, it's on purpose. The achievement is awarded when you have a created repository with >= 16 stars. When you reach 128 stars the bronze tier is unlocked, then silver at 512 and gold at 4096. You can see an example of how it looks with all unlocked here: https://github.com/torvalds?achievement=starstruck&tab=achievements

Note that it still says 16, 128, 512, and 4096 stars even though the repository has more.

**Reply by realamirhe** [reply]
Thanks for your response :)

**Reply by dandv** [reply]
I was also confused by this. The count jumps from 16 to 128, even though I have a repo with 40 stars.

**Reply by lucaspaiva-lp** [reply]
Wow!

**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by 4po** [comment]
That's not wrong, it's just a report of your repostories of the dates where the stars have increased. 
https://github.com/torvalds?achievement=starstruck&tab=achievements

**Reply by Karkatov** [reply]
I will mutually add your repository to favorites, here is mine: https://github.com/Karkatov/TaskManager

**Reply by dandv** [reply]
> I will mutually add your repository to favorites, here is mine: https://github.com/Karkatov/TaskManager

This doesn't feel ethical. @github-staff

**Comment by VictorHugoAguilar** [comment]
Hello all,
for get starstruck need to reach 10 star. If u want u can put star on my readme repository, I will do the same for you.

https://github.com/VictorHugoAguilar/ticketing-microservices-node-nextjs-react

**Reply by dandv** [reply]
> Hello all, for get starstruck need to reach 10 star. If u want u can put star on my readme repository, I will do the same for you.

@github-staff, is it OK to solicit likes for likes?

**Reply by tjcchen** [reply]
@dandv @github-staff Yeah, I think this is inappropriate too.

**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by dripton-art** [comment]
I'm wondering if I put stars on other people's repositories, do I also earn the Starstruck badge? 
@github-staff

**Reply by shaido987** [reply]
No, it's the number of stars on your own repository.

**Reply by dripton-art** [reply]
ohhh thanks alot

**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by khalifarsm** [comment]
https://github.com/khalifarsm/botox-android

**Comment by davidlucus80** [comment]
That error usually means R is using an old or invalid GitHub token. Quick fix: run gitcreds::gitcreds_delete(), then try remotes::install_github("johnbaums/rmaxent") again. If you want a permanent fix, create a new personal access token on GitHub, set it with gitcreds::gitcreds_set(), then the install will work without the 401.

**Comment by YarinsHyper** [comment]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by Gihchathur** [comment]
Hello all,
for get starstruck need to more star. If u want u can put star on my readme repository, I will do the same for you.
https://github.com/Gihchathur/PulseDash
